bot/pagination.py

```diff
@@ -97,6 +155,7 @@ async def paginate(
         suffix: str = "",
         max_lines: t.Optional[int] = None,
         max_size: int = 500,
+        scale_to_size: int = 2000,
```

Comment:
This isn't actually being used anywhere inside the function. I think it's supposed to be used at line 204.

Reply:
Oops, good catch. I had intended to add it there but forgot to.